### DISCUSSION TITLE
Compatible with Electron 21+ / V8's Memory Cage?

### DISCUSSION THREAD

**Discussion by hrueger** [m1dugh/native-sound-mixer#37] [Q&A] [answered]
Hi,
I couldn't find any info on whether this is compatible with the [Memory Cage of Electron 21+](https://www.electronjs.org/blog/v8-memory-cage). Is it?
I just don't want to risk that it works on my machine and users of my application are doing something differently and it crashes for them.

**Comment by m1dugh** [comment]
From what I read, everything in my code should fit the requirements of the memory cage. Nevertheless, I never tested so I can't tell, I'll have to check.

**Comment by hrueger** [comment, marked as answer]
Thanks for the clarification and for the very fast response!
That's probably enough for my use case, I'll make sure to test it intensively once all the dependencies I'm using have been updated (so far that's only 4 of 10...).

I just don't know enough about c++ to skim through the code and check.

Should we leave this issue open until I've tested and verified it? Might take some time, though, for the reason above.
